CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/svg/SVGContainer.java

```diff
@@ -0,0 +1,125 @@
+/*
+* Copyright 2010-2023 Australian Signals Directorate
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+package au.gov.asd.tac.constellation.plugins.importexport.svg;
+
+import au.gov.asd.tac.constellation.plugins.importexport.svg.resources.SVGAttributeConstant;
+
+/**
+ * A wrapper class for SVGObjects.
+ * Represents a non renderable element that serves to provide
+ * a container for elements to base their relative positions from.
+ * Enables the sizing and positioning of an element and all of the 
+ * elements nested within it.
+ * 
+ * @author capricornunicorn123
+ */
+public class SVGContainer {
+    
+    private final SVGObject svgObjectReference;
+    
+    /**
+     * Wrapper class for SVGObjects. 
+     * Represents a non renderable element that contains other elements.
+     * @param svg 
+     */
+    public SVGContainer(final SVGObject svg) {
+        this.svgObjectReference = svg;
+    }
+    
+    /**
+     * Sets the height of the container element.
+     * @param height
+     */
+    private void setHeight(final Float height) {
+        svgObjectReference.setAttribute(SVGAttributeConstant.HEIGHT.getKey(), height.toString());
+    }
```

Comment:
if you wanted to use the primitive `float` here, you could use `String.valueOf(height)`. If null is valid though, note that `height.toString()` throws a null pointer if `height` is null so you'll need to add a null check there. Will be a similar case for the other similar functions